indico/modules/events/templates/header.html

```diff
@@ -1,31 +1,31 @@
 {%- macro _render_nav_bar() -%}
     {% set rel = event.get_relative_event_ids() %}
 
-    <a class="i-button icon-home" href="{{ url_for_index() }}"
+    <a class="i-button text-color subtle icon-home" href="{{ url_for_index() }}"
```

Comment:
I'd rather import these two classes in the `.button-bar` `.i-button`  to keep everything in one place